Documentation/migration-guide.md

```diff
@@ -148,6 +148,25 @@ MobileCore.registerExtensions([
 
 ------
 
+### Granular Ad Tracking
+
+AEPMedia allowed granular ad content tracking to `1 second` from `default 10 seconds` by setting the `MediaConstants.MediaObjectKey.GRANULAR_AD_TRACKING` in the media object. AEPEdgeMedia is even more customizable and now the ad content tracking interval can be set using the tracker configuration to a value between `[1-10] seconds`. For more information refer [createTrackerWithConfig API](api-reference.md/#createTrackerWithConfig).
```

Comment:
- ... allowed ad content tracking of 1 second when setting the ... (we can skip the default part here)
- For more details, refer to the ..